example/Gemfile

```
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
+ruby ">= 2.6.10"
+
+# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
+# bound in the template on Cocoapods with next React Native release.
+gem 'cocoapods', '>= 1.13', '< 1.15'
+gem 'activesupport', '>= 6.1.7.5', '< 7.1.0'
```

RuboCop (1.81.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `activesupport` should appear before `cocoapods`.

(Bundler/OrderedGems)